cmd/libs/go2idl/client-gen/generators/client-generator.go

```diff
@@ -32,10 +32,16 @@ import (
 
 // NameSystems returns the name system used by the generators in this package.
 func NameSystems() namer.NameSystems {
+	pluralExceptions := map[string]string{
+		"Endpoints":       "endpoints",
+		"ComponentStatus": "componentStatus",
```

Comment:
those looks a bit strange - can we have both capitalized or both not-capitalized?